2/debian-10/rootfs/opt/bitnami/scripts/libkafka.sh

```diff
@@ -93,9 +94,9 @@ kafka_env() {
 export KAFKA_BASEDIR="/opt/bitnami/kafka"
 export KAFKA_HOME="$KAFKA_BASEDIR"
 export KAFKA_LOGDIR="$KAFKA_BASEDIR"/logs
-export KAFKA_CONFDIR="$KAFKA_BASEDIR"/conf
-export KAFKA_CONF_FILE_NAME="server.properties"
-export KAFKA_CONF_FILE="$KAFKA_CONFDIR"/$KAFKA_CONF_FILE_NAME
+export KAFKA_CONFDIR="$KAFKA_BASEDIR"/config
+export KAFKA_CONF_FILE="$KAFKA_CONFDIR"/server.properties
+export KAFKA_MOUNTED_CONFDIR="${KAFKA_MOUNTED_CONFDIR:-${KAFKA_VOLUMEDIR}/config}"
 export KAFKA_VOLUMEDIR="/bitnami/kafka"
```

Comment:
The `KAFKA_MOUNTED_CONFDIR` is defaulting to `KAFKA_VOLUMEDIR`, which is defined afterwards, which will not work.

```suggestion
export KAFKA_VOLUMEDIR="/bitnami/kafka" 
export KAFKA_MOUNTED_CONFDIR="${KAFKA_MOUNTED_CONFDIR:-${KAFKA_VOLUMEDIR}/config}"
```